frame/oracle/src/lib.rs

```diff
@@ -721,7 +739,7 @@ pub mod pallet {
 						slash_amount,
 					));
 				} else {
-					let reward_amount = asset_info.reward;
+					let reward_amount = Self::possible_reward(asset_info.reward);
```

Comment:
Everything is looking fine, but I have one question: right now if the reward of an asset exhaust the block reward, the remaining oracles won't get rewared. Would it be better to divide the "penalty" over all the rewards for the current block?